backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomsheet/internal/BpkModalBottomSheetImpl.kt

```diff
@@ -170,7 +177,15 @@ private fun BpkModalBottomSheetHeader(
                     text = it,
                     style = BpkTheme.typography.heading5,
                     textAlign = TextAlign.Center,
-                    modifier = Modifier.semantics { heading() },
+                    modifier = Modifier
+                        .semantics {
+                            heading()
+                        }
+                        .applyIf(titleContentDescription != null) {
+                            semantics {
+                                contentDescription = titleContentDescription ?: ""
```

Comment:
small one, but `titleContentDescription` should not be null at this point

Reply:
Yes true, but the editor was complaining since the null check is not in context, okay to replace with `!!` ?

Reply:
let's use `!!` like we use in other similar situations like https://github.com/Skyscanner/backpack-android/blob/055475462b6d645476467fe9a151b2239638c8bd/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/internal/BpkButtonImpl.kt#L93
if at some point the compiler turns smarter we'll just need to remove those and we do not need to define empty strings as default